WORKING_GROUPS.md

```diff
@@ -0,0 +1,247 @@
+# Node.js Working Groups
+
+Node.js Working Groups are autonomous projects created by the
+[Technical Steering Committee (TSC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#technical-committee).
+
+Working Groups can be formed at any time but must be ratified by the TSC.
+Once formed the work defined in the Working Group charter is the
+responsibility of the WG rather than the TSC.
+
+It is important that Working Groups are not formed pre-maturely. Working
+Groups are not formed to *begin* a set of tasks but instead are formed
+once that work is already underway and the contributors
+think it would benefit from being done as an autonomous project.
+
+If the work defined in a Working Group charter is completed the Working
+Group should be dissolved and the responsibility for governance absorbed
+back in to the TSC.
```

Comment:
@ashleygwilliams ... perhaps slight modification to, `If the work defined in a Working Group charter is completed, the Working Group should be dissolved and the responsibility for governance for governance absorbed back into the TSC. A Working Group can be dissolved either through consensus of the Working Group membership or normal TSC motion and vote`